### DISCUSSION TITLE
Question regarding  "copy as an image" button

### DISCUSSION THREAD

**Discussion by just-a-garlic-bread** [jgraph/drawio-desktop#1314] [Q&A]
How can i extract/copy an image from an already existing file of a diagramm?
I tried searching for answers oh google and official q&a, tried using "copy" and "copy as an image"(both work the same way, i presume, just copying it as a part of diagram and not as a file) buttons, but to no result
Is it possible in some way?
It's not that important, or urgent, but still would be really cool if there's a way to do so

**Comment by ezimore** [comment]
You can use Copy as Image command from Edit menu.
If no element selected, it creates an image from the whole diagram of the actual sheet.
If elements selected, image contains only those elements.

Trick: create white empty rectangulars to set the image size. E.g. having a vertical diagram, creates 2 white rectangulars horizontally and copy image creates a nice image which do not need to resize in word.

**Comment by just-a-garlic-bread** [comment]
It seems like the app was updated, fixing an issue(i was unable to paste it anywhere before)
Now (and i dont want to seem too needy), it , somehow, creates a black(or the colour of the background, i suppose) border around an image
I mean, its not that serious, but still good to notice that it does so
(Another thing to note:The border doesnt appear when pasting the image back into the app)
![example](https://github.com/jgraph/drawio-desktop/assets/132960873/30269147-3fea-459c-994d-67bf79f49d9a)
